### PR TITLE
Revise coe uk quick start

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-d77326d67f/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f6823d65be/how-to/)

**Comment by cyrichardson** [comment]
@DonNateR Edits in this Word doc:
[cloud-office-quick-start-for-uk-customers.docx](https://github.com/rackerlabs/rackspace-how-to/files/1661573/cloud-office-quick-start-for-uk-customers.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d90c370531/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-fe8b46dcc1/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9c523d2203/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-501751cb0c/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-17764e0af6/how-to/).

File: content/rackspace-email/cloud-office-quick-start-for-uk-customers.md [hunk 1]
```diff
@@ -1,119 +1,79 @@
...
-    click **Mailboxes**.
-
-5.  Click the domain to which you want to add mailboxes.
+### Add mailboxes
 
-6.  Click **Add Mailbox**.
+Follow these steps to add a mailbox.
 
-7.  On the **General** tab, enter values in the User Details section.
+1.  After accessing your Cloud Office Control Panel, in the email hosting section click **Mailboxes**.
```
Comment (catlook) [inline content/rackspace-email/cloud-office-quick-start-for-uk-customers.md]: ... section, click ...

File: content/rackspace-email/cloud-office-quick-start-for-uk-customers.md [hunk 2]
```diff
@@ -1,119 +1,79 @@
...
-registrar and have them update your MX records using the following
-values:
+If you do not have access to your DNS records, contact your domain registrar and have them update your MX records by using the following values:
 
 -   `MX1.EMAILSRVR.COM (Priority 10)`
 -   `MX2.EMAILSRVR.COM (Priority 20)`
 
-The MX record changes take about 24 hours to update. You will know when
-the update has occurred because mail will start being delivered to your
+The MX record changes take about 24 hours to update. During the update mail starts being delivered to your
```
Comment (catlook) [inline content/rackspace-email/cloud-office-quick-start-for-uk-customers.md]: ... the update, mail starts ...

File: content/rackspace-email/cloud-office-quick-start-for-uk-customers.md [hunk 3]
```diff
@@ -122,94 +82,57 @@ Panel](https://cp.rackspace.com/Login.aspx?ReturnUrl=Default.aspx).
...
 5.  Enter the new alias in the **Create New Alias** field.
-
-6.  *(Optional)* Select members of your domain and move them to the
-    **Members of this Alias** field, and click **Add**.
-
-7.  *(Optional)* Add email addresses for members outside of your domain
-    into the **Email Address** field and click **Add**.
-
+6.  *(Optional)* Select members of your domain and move them to the **Members of this Alias** field, and click **Add**.
+7.  *(Optional)* Add email addresses for members outside of your domain into the **Email Address** field and click **Add**.
```
Comment (catlook) [inline content/rackspace-email/cloud-office-quick-start-for-uk-customers.md]: .. field, and  ...

File: content/rackspace-email/cloud-office-quick-start-for-uk-customers.md [hunk 4]
```diff
@@ -122,94 +82,57 @@ Panel](https://cp.rackspace.com/Login.aspx?ReturnUrl=Default.aspx).
...
+1.  Log in to the [MyRackspace Customer Portal](https://my.rackspace.com/).
 2.  Select **Account &gt; Payments** from the drop-down menu.
 
 ### View and download invoices
 
 To view and download your invoices follow these steps:
 
 1.  Log in to [MyRackspace Customer Portal](https://my.rackspace.com).
-
 2.  Click **Account** and select **Transactions**.
```
Comment (catlook) [inline content/rackspace-email/cloud-office-quick-start-for-uk-customers.md]: ... **Account**, and select ...

File: content/rackspace-email/cloud-office-quick-start-for-uk-customers.md [hunk 5]
```diff
@@ -122,94 +82,57 @@ Panel](https://cp.rackspace.com/Login.aspx?ReturnUrl=Default.aspx).
...
 
     You can view an invoice by clicking on its number.
 
-3.  To download your invoices, click the **Actions** button and select
-    **Export All Transactions**.
+3.  To download your invoices, click the **Actions** button and select **Export All Transactions**.
```
Comment (catlook) [inline content/rackspace-email/cloud-office-quick-start-for-uk-customers.md]: ... button, and select ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-470ceff76d/how-to/).

**Pull request merged**